src/ImageIO/CGImageAnimation.cs

```diff
@@ -0,0 +1,128 @@
+//
+// ImageIO - CGImageAnimation.cs
+//
+// Authors:
+//	Whitney Schmidt  <whschm@microsoft.com>
+//
+// Copyright 2020, Microsoft Corp.
+//
+
+using System;
+using System.Runtime.InteropServices;
+using CoreFoundation;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace ImageIO {
+
+    public partial class CGImageAnimation {
+
+        public delegate void CGImageSourceAnimationBlock (nint arg0, CGImage arg1, out bool done /*check*/ );
```

Comment:
According to the documentation, the parameters should be named:

```suggestion
        public delegate void CGImageSourceAnimationBlock (nint index, CGImage image, out bool stop /*check*/ );
```